### ISSUE TITLE
Hyperparameters used

### ISSUE THREAD

**Issue by Srinivasa-Divakar-Bhat** [ShadeAlsha/LTR-weight-balancing#9]
Hi,
Interesting work here!
Can you please provide the details on the following hyperparameters used for each of the following datasets,
INaturalist18 : weight decay, learning rate used. 
ImagenetLT: weight decay, learning rate used.

Thank you in advance!

**Comment by xiaohe6** [comment]
Hi,
Take the liberty of interrupting！
Have you now found the weight decay and lr parameters?
